### DISCUSSION TITLE
short connection lead to too much time_wait

### DISCUSSION THREAD

**Discussion by songlong** [bfenetworks/bfe#918] [Q&A (问题和回答)] [answered]
when use short connection as upstream proxy 
by set MaxIdleConnsPerHost=0

but I found too much time_wait?

not like go http reqeust, it must call response.Body.Close() to close http reqeust client.
but i can not see it configure?

**Comment by iyangsj** [comment, marked as answer]
The **Close()** will be called when the response has been forwarded to client.

See: https://github.com/baidu/bfe/blob/develop/bfe_server/reverseproxy.go#L679   @songlong

**Comment by songlong** [comment]
old version closed not like this.
so it can slove too much time_wait?
